clang/lib/CIR/CodeGen/CIRGenVTables.cpp

```diff
@@ -622,8 +622,7 @@ void CIRGenVTables::emitVTTDefinition(cir::GlobalOp VTT,
              "Did not find ctor vtable address point!");
     }
 
-    mlir::Attribute Idxs[3] = {
-        CGM.getBuilder().getI32IntegerAttr(0),
+    mlir::Attribute Idxs[2] = {
```

Comment:
The idea don't create zero index here and add it in the lowering. Otherwise the lowering part would be too ugly on my taste

Reply:
This sounds fine, can we add a note in some of the tablegen op docs to highlight this expectation for the users? (e.g. CIR computes this differently from LLVM since ...)

Reply:
done! Have no idea what the best place for the comment though